storybook/.storybook/main.js

```diff
@@ -66,17 +66,39 @@ module.exports = Object.assign(({ args }) => {
         {
           test: /\.(tsx?)$/,
           exclude: /node_modules/,
-          loader: require.resolve('@sucrase/webpack-loader'),
+          loader: require.resolve('swc-loader'),
           options: {
-            transforms: ['typescript', 'jsx', 'react-hot-loader'],
+            jsc: {
```

Comment:
Might as well set target here as well right? Just making it a bit smaller and faster